etc/tendrl/snmp.conf.yaml.sample

```diff
@@ -0,0 +1,26 @@
+# V2 end point details
+# Enable the bvelow lines if v2 end point to be used for SNKP notifications
+# Nopte: for different communities make different similar entries below additionally
+# Here default community is public, it is also modifiable
+# V2 is auth less 
+
+# v2_endpoint:
```

Comment:
Put `#` at the first column for all the commented lines. Its easier to remove later for a block :)